orders-war/src/test/java/au/gov/qld/pub/orders/scenario/ScenarioSetup.java

```diff
@@ -1,64 +1,72 @@
 package au.gov.qld.pub.orders.scenario;
 
+import java.lang.invoke.MethodHandles;
+import java.lang.management.ManagementFactory;
 import java.net.URL;
+import java.time.Duration;
 
+import au.gov.qld.online.selenium.DriverTypes;
+import au.gov.qld.online.selenium.SeleniumHelper;
+import au.gov.qld.online.selenium.WebDriverHolder;
 import com.dumbster.smtp.SimpleSmtpServer;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.BeforeClass;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.htmlunit.HtmlUnitDriver;
 
 import au.gov.qld.pub.orders.scenario.selenium.ConfirmPage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ScenarioSetup {
 
+    private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
     public static final String BASE_URL = "http://localhost:8091/";
-    private static boolean embedded = false;
 
-    public static WebDriver driver;
+    protected WebDriverHolder webDriverHolder;
+
     private static SimpleSmtpServer mailServer;
 
-    @BeforeClass
-    public static void setupDriver() {
-        driver = new HtmlUnitDriver(false);
-    }
 
-    @AfterClass
-    public static void teardownDriver() {
-        driver.quit();
+    @BeforeAll
+    public static void startMailServer() {
+        try {
+            System.out.println("Starting embedded instance");
+            mailServer = SimpleSmtpServer.start(1325); //ensure same in application.properties spring.mail.port
```

Comment:
"Ensure same" - could this, should this be a variable or function to ensure it is indeed the same?

Reply:
leaving as be this is mostly a poc project and once we find a fix for our other ssq apps we can do that here.